server/graphql/dog-licenses.js

```diff
@@ -52,69 +74,120 @@ type LicensesArgs = {
   ids: string[],
 };
 
+//must match graphql schema above
```

Comment:
Remember that GraphQL types are nullable by default, but Flow types are not. If something doesn’t have a `!` in the GraphQL then it should start with a `?` here. (_E.g._ `?string')